tests/edge-to-edge/server/app.spec.ts

```diff
@@ -202,11 +207,12 @@ describe( 'Termbox SSR', () => {
 	it( 'renders the termbox when requesting /termbox with well-formed query for known entity', ( done ) => {
 		const entityId = 'Q64';
 		const language = 'de';
+		const editLink = '/some';
 
 		nockSuccessfulLanguageLoading( language );
 		nockSuccessfulEntityLoading( entityId );
 
-		request( app ).get( '/termbox' ).query( { entity: entityId, language } ).then( ( response ) => {
+		request( app ).get( '/termbox' ).query( { entity: entityId, language, editLink } ).then( ( response ) => {
 			expect( response.status ).toBe( HttpStatus.OK );
 
 			const $dom = getDomFromMarkup( response.text );
```

Comment:
Should we not assert that the link appears in the mark-up?

Reply:
yes! done